content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+## Proposal {.page_break_before}
+
+Manubot is a system for writing scholarly manuscripts on GitHub.
+Manubot aims to **transform publishing** to be transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
+Wooh!
+
+Manubot applies **tactics from open source** software to manuscript writing.
+Manuscripts are written in markdown, using git for version control and GitHub for collaboration and review.
+Continuous integration automatically rebuilds and deploys the manuscript when it changes [@doi:10.1038/nbt.3780; @doi:10.1038/550143a].
+Manubot provides many state-of-the-art features.
+For example, users cite standard identifiers, while bibliographic details are automatically retrieved.
+
+Currently, the Manubot consists of two repositories: [`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) (a blank manuscript with the required files) and [`greenelab/manubot`](https://github.com/greenelab/manubot) (a backend Python package).
+As of December 2017, [10](https://github.com/greenelab/manubot-rootstock/graphs/contributors) individuals have contributed to the **Manubot codebase**, which has [33](https://github.com/greenelab/manubot-rootstock/stargazers) ⭐s on GitHub.
+For more information on the Manubot, see [these slides](http://slides.com/dhimmel/manubot#/) as well as the in-progress [Meta Review](https://greenelab.github.io/meta-review/).
+
+We initially created the Manubot to enable the **Deep Review**, a review paper on deep learning that we collaboratively drafted [on GitHub](https://github.com/greenelab/deep-review).
+Over 30 authors contributed to the Deep Review [@doi:10.1101/142760], using [issues](https://github.com/greenelab/deep-review/issues?utf8=%E2%9C%93&q=is%3Aissue) to discuss source material and [pull requests](https://github.com/greenelab/deep-review/pulls?utf8=%E2%9C%93&q=is%3Apr) to propose manuscript changes.
+The community appreciated the breadth of perspectives on a quickly-developing topic and the Deep Review was the most-viewed _bioRxiv_ preprint from April–June 2017 [@doi:10.1038/d41586-017-08493-x].
+We've also used the Manubot to write our [Sci-Hub Coverage Study](https://greenelab.github.io/scihub-manuscript/) and to reproduce the [Bitcoin Whitepaper](https://dhimmel.github.io/bitcoin-whitepaper/) [@tag:steemit-post].
```

Comment:
```
The Sci-Hub coverage study preprint is the second most accessed PeerJ preprint ever (link jordan's resource).
In its first year the collaborative, open, and review-driven manubot authoring process was employed for two of the most accessed preprints of the year.
```
☝️ not fully tied to this, but it's worth noting that I think manubot also helps contributions be better (not just easier to write/update/mine computationally).

Reply:
Addressed in https://github.com/greenelab/manufund-2018/pull/2/commits/26d1a2eedd4291fbb08cfeab337059f012ad9bce